back-end/tests/models/user-quests.test.js

```diff
@@ -0,0 +1,54 @@
+const User = require("../../db/models/users");
+const Quest = require("../../db/models/quests");
+const faker = require("faker");
+// sequelizeClient.sync({ force: true, match: /_test$/ });
+
+describe("user relationships", () => {
+  it("can fetch the related quests", async () => {
+    // data creation
+    const villager = User.create({
+      username: faker.internet.userName(),
+      firstName: faker.name.firstName(),
+      lastName: faker.name.lastName(),
+      email: faker.internet.email(),
+      password: faker.internet.password(),
+      avatar: faker.internet.avatar(),
+      adventurer: false,
+      bio: faker.lorem.paragraph(),
+    });
+
+    const adventurer = User.create({
+      username: faker.internet.userName(),
+      firstName: faker.name.firstName(),
+      lastName: faker.name.lastName(),
+      email: faker.internet.email(),
+      password: faker.internet.password(),
+      avatar: faker.internet.avatar(),
+      adventurer: true,
+      bio: faker.lorem.paragraph(),
+    });
+
+    Quest.create({
+      name: faker.name.jobType(),
+      description: faker.lorem.paragraph(),
+      completed: false,
+      city: faker.address.city(),
+      classId: 1,
+      adventurerId: adventurer.id,
+      villagerId: villager.id,
+    });
+
+    Quest.create({
+      name: faker.name.jobType(),
+      description: faker.lorem.paragraph(),
+      completed: false,
+      city: faker.address.city(),
+      classId: 1,
+      villagerId: villager.id,
+    });
+    // test
+    const quests = User.findByPk(1).getQuests();
```

Comment:
to make things simpler, I would probably get rid of the quest for the villager, and create another quest for that same adventurer you created. After which you could do:
```suggestion
    const adventurerQuests = User.findByPk(adventurer.id).getQuests();
```
and the `expect` below should see the 2 quests for that user.

As an alternative, you could write another scenario (using another `it()`).